componentsPage.html

```diff
@@ -214,6 +215,55 @@ <h2>Icon Buttons</h2>
             input Will Come Here...
            </section>
 
+           <section class="section" id="textUtilities-container">
+               <h1>Text Utilities</h1>
+               <h2>Heading</h2>
+               <p>Our Component library has all headings from <span>h1</span> to <span>h6</span></p>
+               <div class="headings-container">
+                   <h1 class="heading">H1 - Heading</h1>
+                   <h2>H2 - Heading</h2>
+                   <h3>H3 - Heading</h3>
+                   <h4>H4 - Heading</h4>
+                   <h5>H5 - Heading</h5>
+                   <h6>H6 - Heading</h6>
```

Comment:
Hey Omkar, I have a suggestion like to have separate class names for different headings.